doc/man3/SSL_get_handshake_rtt.pod

```diff
@@ -0,0 +1,64 @@
+=pod
+
+=head1 NAME
+
+SSL_get_handshake_rtt
+- get round trip time for SSL Handshake
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ uint64_t SSL_get_handshake_rtt(const SSL *s);
+
+=head1 DESCRIPTION
+
+SSL_get_handshake_rtt() returns the round trip time for I<ssl>,
+for both he client and a server. If I<ssl> is NULL, 0
+is returned.
+
+This metric is returned in microseconds (us) as a uint64_t data type.
```

Comment:
No, this hasn't been discussed yet. Could you elaborate on what that might look like? 

If its easiest, we could just return `-1` in the case of failure so that any programs calling `SSL_get_handshake_rtt()` don't mistakenly divide by zero.

Reply:
Nope, the suggested signature is: `int SSL_get_handshake_rtt(const SSL *ssl, uint64_t *rtt);` With return value being just 0 for failure 1 for success and rtt being the output parameter.

Reply:
t8m got what I was thinking about, thanks.
I think I remember a previous discussion among the team about when it makes sense to just return the output value directly vs returning success/failure with a separate output variable, and I was hoping another member of the team would remember more details about what criteria we landed on.  (Of course, I might be remembering a discussion from a different project...)

Reply:
This makes sense and I've pushed this refactor. Thanks for clarifying